components/brave_wallet/resources/BUILD.gn

```diff
@@ -0,0 +1,18 @@
+import("//brave/components/common/typescript.gni")
+import("//tools/grit/grit_rule.gni")
+import("//tools/grit/repack.gni")
+
+transpile_web_ui("brave_wallet_script_resources") {
+  entry_points = [ [
+        "brave_wallet_script",
+        rebase_path("brave_wallet_provider.js"),
+      ] ]
+
+  resource_name = "brave_wallet_script"
+}
+
+pack_web_resources("generated_resources") {
```

Comment:
This is a bit confusing with generated_resources for brave_wallet for the panel and the page. 
Maybe call it `ethereum_provider_generated_resources`?

See
https://github.com/brave/brave-core/blob/master/components/brave_wallet_ui/BUILD.gn#L29

Mainly brave_wallet is not specific enough because it is used for a lot of things. We should say what it is specifically.  Which is an ethereum provider.

Reply:
corrected